examples/LXHeadless/src/heronarts/lx/headless/LXHeadless.java

```diff
@@ -35,16 +32,16 @@
 
   public static LXModel buildModel() {
     // TODO: implement code that loads and builds your model here
-    return new GridModel(30, 30);
+    return new GridModel(10,17);
```

Comment:
Before changing this, I'd love to see what the IndexOutOfBounds exception was that was being caused by 30, 30. That's concerning, implies that some size limit was being improperly hardcoded elsewhere...